utils/gyb.py

```diff
@@ -655,7 +656,9 @@ def execute(self, context):
         # Execute the code with our __children__ in scope 
         context.localBindings['__children__'] = self.children
         result = eval(self.code, context.localBindings)
-        assert context.localBindings['__children__'] is self.children
+
+        if context.localBindings['__children__'] is not self.children:
+            raise ValueError("The code is not allowed to mutate children")
```

Comment:
Running the test fails on the master branch.

```bash
./gyb --test gyb.py
```
Unless I'm running the tests incorrectly? How can I make the tests pass on master so I can add my tests?